### PR TITLE
Removed --password from all validator commands

### PR DESCRIPTION
**Pull request by wgknowles**
The use of the --password flag should not be reccommended as this may result in passwords being logged to command history 
related #36

### CODE REVIEW

**Comment by wgknowles** [comment]
closes prysm-testnet-site/issues/#113

File: docs/install/lin/activating-a-validator.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The wallet is scanned for the required amount of Göerli ETH after being linked.
 
 Depending on your platform, issue the appropriate command from the examples below to generate a public / private keypair for your validator.
 
+> NOTICE: When prompted, provide a password to encrypt your new ETH2 validator and withdrawl keys.
```
Comment (har00ga) [inline docs/install/lin/activating-a-validator.md:24]: Please ensure you match formatting with these, notice is always bolded (ie ` > **NOTICE:** words`

Comment (wgknowles) [reply docs/install/lin/activating-a-validator.md]: fixed across all files

**Comment by har00ga** [comment]
Thanks :)

**Pull request merged**